src/sentry/static/sentry/less/includes/bootstrap/forms.less

```diff
@@ -45,7 +45,7 @@ label {
 
 input[type='search'] {
   // Override content-box in Normalize (* isn't specific enough)
-  .box-sizing(border-box);
+  box-sizing: border-box;
```

Comment:
I’m guessing that 90% of this code can be removed entirely but for now it stays.